website/docs/docs/deploy/continuous-deployment.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Continuous deployment in dbt Cloud"
+sidebar_label: "Continuous deployment"
+description: "Learn about continuous deployment (CD) workflows "
+---
+
+To help you improve data transformations and to ensure up-to-date data products, you can run [merge jobs](/docs/deploy/merge-jobs) to implement a continuous deployment (CD) workflow in dbt Cloud. Merge jobs can automatically build modified models for you whenever a pull request (PR) merges, making sure the latest changes are in production (such as modifications to code that generates the tables). You don't need to wait for the next scheduled job to run. 
```

Comment:
Should we use the dbt marketing line here and say "ship data products faster" instead of "ensure up-to-date data products"?